### ISSUE TITLE
Recommend Roarr over Pino

### ISSUE THREAD

**Issue by gajus** [nodeshift/nodejs-reference-architecture#69]
The current recommended component for logging is Pino

https://github.com/nodeshift/nodejs-reference-architecture/blob/main/docs/operations/logging.md

[Roarr](https://github.com/gajus/roarr) is an actively maintained Node.js JSON logger that has several benefits over Pino and other frameworks:

* Does not block the event cycle (=fast).
* Does not require initialisation.
* Produces structured data.
* Decouples transports.
* Has a CLI program and [web application](https://roarr.io/) for reading logs.
* Works in Node.js and browser.
* Configurable using environment variables.

I have provided an in depth write up about my [motivation](https://github.com/gajus/roarr#roarr-motivation) for creating and maintaining Roarr.

Perhaps the most useful (unique) feature of Roarr is [`adopt`](https://github.com/gajus/roarr#adopt), which allows to inherit log context and describes relationships between logs in async contexts using [`sequence`](https://github.com/gajus/roarr#sequence-value).

**Comment by mhdawson** [comment]
@gajus thanks for the heads up on another option for logging. 

One of the key elements of the recommendations is that they are based on the existing experience in our internal deployments and work with our customers. From https://github.com/nodeshift/nodejs-reference-architecture/blob/main/README.md, the key aspect we've documented is `Where possible the opinion is based on what we've used internally and in our customer engagements.`

We acknowledge that there will be other options which are as good but the net is that we'll end up pointing to ones that we have experience with.

At this point we have a number of internal teams/implementations using pino and have experience with it across the different groups which is one of the reasons it is in the architecture.

I've leave this issue open for another week for visibility across the team, but to set expectations, we'd have to see our internal usage to shift from pino to Roarr before we are likely to update our suggestions.
